### DISCUSSION TITLE
Is it possible to remove StackDivider's margins?

### DISCUSSION THREAD

**Discussion by runoncedev** [chakra-ui/chakra-ui#6144] [Q&A] [answered]
Trying to remove margins from [` StackDivider`](https://chakra-ui.com/docs/components/layout/stack#stack-dividers):

```
<Stack divider={<StackDivider p="0" m="0" />}>
  <p>Hello</p>
  <p>World</p>
</Stack>
```

Also tried using other margin/padding props and curly braces:

```
<StackDivider pt={0} mt={0} />
```

But for some reason it won't work, the padding and margin are not being applied to the element.

Using `@chakra-ui/react@2.1.2`.

**Comment by aroyan** [comment, marked as answer]
I don't think it's possible. I've inspect `<StackDivider />` component in React Dev tools and it has two props `borderColor` and `__css`, but I've tried to pass `__css` props into `<StackDivider />` but when I hover to that props it's show description that `__css` props is used for internal css management @private so I think that props is only can be access by Chakra-ui teams. And why passing margin and padding props to 0 doesn't worked, because the margin is not set directly into `<StackDivider />` component, as you can see in react dev tools on image below
![image](https://user-images.githubusercontent.com/43630681/171990247-751bbf6e-42cb-49a0-a91a-35fa724367b7.png)

**Reply by runoncedev** [reply]
Aw, that's a bummer! I guess I can create my own divider then. Thank you!

**Reply by aroyan** [reply]
I think I've found a way to remove those margins by using `!important` CSS property. Here sandbox I've created https://codesandbox.io/s/xenodochial-mcclintock-ijimi0?file=/src/App.js

**Reply by runoncedev** [reply]
Cool! But having to use `!important` is not very nice.

**Reply by runoncedev** [reply]
Ok, creating my own divider using Chakra components doesn't work either 😢 

The margin is applied and can't be removed either.

**Reply by runoncedev** [reply]
Just realized you can control the space between items using the `spacing` prop. I was using `gap` instead, my bad! 😅
